### PR TITLE
Image cleanup

### PR DESCRIPTION
**Pull request by gklopper**
Deleting old image service code now that we have moved to the new service.

This **cannot be merged** till some config has rolled out for our CI servers.

Likewise devs now need the following config (it was optional till now)

```
images.media.token=XXX
images.static.token=YYY
```

### CODE REVIEW

File: common/app/views/support/Profile.scala [hunk 1]
```diff
@@ -88,24 +88,20 @@ object ImgSrc extends Logging {
...
       val uri = new URI(url.trim)
 
-      val supportedImages = if (PngResizingSwitch.isSwitchedOn) Set(".jpg", ".jpeg", ".png") else Set(".jpg", ".jpeg")
-
       val isSupportedImage = supportedImages.exists(extension => uri.getPath.toLowerCase.endsWith(extension))
 
       hostPrefixMapping.get(uri.getHost)
-        .filter(_ => isSupportedImage)
-        .filter(_ => ImageServerSwitch.isSwitchedOn)
+        .filter(const(ImageServerSwitch.isSwitchedOn))
```
Comment (johnduffell) [inline common/app/views/support/Profile.scala]: I learned something new about scala just then

File: png-resizer/app/controllers/Resizer.scala [hunk 2]
```diff
@@ -1,135 +0,0 @@
...
-  }
-
-  def resize(backend: String, path: String, widthString: String, qualityString: String) =
-    Action.async { request =>
-
-      (Backends.uri(backend, path), widthString, qualityString) match {
-        case (Some(uri), IntString(width), IntString(quality)) =>
-
-          // Left here is the http result, right is the data that still needs computation
-          val resultEitherT = for {
```
Comment (johnduffell) [inline png-resizer/app/controllers/Resizer.scala]: I knew I'd never get away with this ;)

**Comment by johnduffell** [comment]
presumably there's another PR to remove the groups etc from AWS
other than that, :+1:

**Comment by gklopper** [comment]
Yeah, once this is all gone I need to remove the backends in the CDN then the load balancers

File: common/app/views/support/Profile.scala (hunk 1 above)
Comment (gklopper) [inline common/app/views/support/Profile.scala]: I still have not decided which of the two I prefer...

**Comment by johnduffell** [comment]
is this going out soon? consider closing and reopening when it's ready?

File: common/app/views/support/Profile.scala (hunk 1 above)
Comment (kelvin-chappell) [inline common/app/views/support/Profile.scala]:
```
val signedUrl = for {    
  host <- hostPrefixMapping.get(uri.getHost)    
  if ImageServerSwitch.isSwitchedOn    
  if isSupportedImage    
} yield {    
  val signedPath = ImageUrlSigner.sign(s"${uri.getPath}${imageType.resizeString}", host.token)    
  s"$imageHost/img/${host.prefix}$signedPath"    
}    
signedUrl getOrElse url
```

Comment (johnduffell) [inline common/app/views/support/Profile.scala]: nice, very clear @kelvin-chappell

**Pull request merged**